### PR TITLE
konvoy: add a Warning about removal of Argo

### PR DESCRIPTION
**Pull request by dkoshkin**
## Jira Ticket

https://jira.d2iq.com/browse/COPS-7146

## Description of changes being made

Add a Warning in the Konvoy release docs about the removal of Argo CD.

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Create new PRs against `develop`, or `main` for hotfixes to production.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/konvoy/1.8/release-notes/1.8.4/index.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ enterprise: false
 
 <p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. For new customers, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download Konvoy.</p>
 
+<p class="message--warning"><strong>WARNING: </strong>This version includes an update to D2iQ Dispatch that removes Argo CD. Do NOT update the version of Dispatch if you are currently relying on Argo CD that is part of Dispatch. D2iQ Dispatch has been deprecated in favor of Flux. See the <a href="https://d2iq.com/blog/goodbye-dispatch-hello-fluxcd">D2iQ blog post</a> for more information.</p>
```
Comment (cmastr) [inline pages/dkp/konvoy/1.8/release-notes/1.8.4/index.md:18]:
```suggestion
<p class="message--warning"><strong>WARNING: </strong>Konvoy 1.8.4 includes an update to D2iQ Dispatch that removes Argo CD. Do NOT apply the Konvoy 1.8.4 patch if you rely on Argo CD that is part of Dispatch. D2iQ Dispatch has been deprecated in favor of Flux. See the <a href="https://d2iq.com/blog/goodbye-dispatch-hello-fluxcd">D2iQ blog post</a> for more information.</p>
```

Comment (dkoshkin) [reply pages/dkp/konvoy/1.8/release-notes/1.8.4/index.md:18]: Its possible to upgrade the version of Konvoy, Kubernetes-Base-Addons, Kommander without bumping Dispatch.
Im not sure if we want to block users from completely upgrading.

**Review by cneth — APPROVED** [review]

File: pages/dkp/konvoy/1.8/release-notes/1.8.4/index.md (hunk 1 above)
Comment (cmastr) [reply pages/dkp/konvoy/1.8/release-notes/1.8.4/index.md:18]: We discussed this in the bug meeting, and that's what we said but let's check with PM. @sonali-mishra-22 or @chessahoekstra

Comment (cneth) [reply pages/dkp/konvoy/1.8/release-notes/1.8.4/index.md:18]: Dispatch is provided by the following addon:

    - configRepository: https://github.com/mesosphere/kubeaddons-dispatch
      configVersion: stable-1.20-1.4.6

The stable-1.20-1.4.6 version of kubeaddons-dispatch is what was shipped with Konvoy 1.8.4, and this is what removes the ArgoCD tool.   As Dmitri says, it is possible to upgrade  Konvoy, KBA and Kommander WITHOUT upgrading the dispatch addon.   So this is what we should tell people.   Here's my suggestion:

```suggestion
<p class="message--warning"><strong>WARNING: </strong>This version includes an update to the D2iQ Dispatch addon (stable-1.20-1.4.6) that removes Argo CD. Do NOT update to this version of Dispatch if you are currently relying on Argo CD that is part of Dispatch, instead, ensure that the  D2iQ Dispatch version in your config.yaml is stable-1.20-1.4.5.   D2iQ Dispatch has been deprecated in favor of Flux. See the <a href="https://d2iq.com/blog/goodbye-dispatch-hello-fluxcd">D2iQ blog post</a> for more information.</p>
```

Comment (chessahoekstra) [reply pages/dkp/konvoy/1.8/release-notes/1.8.4/index.md:18]: If approved by support, this LGTM.

Comment (sonali-mishra-22) [reply pages/dkp/konvoy/1.8/release-notes/1.8.4/index.md:18]: Looks good to me

**Pull request merged**